cmd/sharded-test-server/main.go

```diff
@@ -32,7 +32,9 @@ import (
 )
 
 func main() {
+	var numberOfShards int
 	flag.String("log-dir-path", "", "Path to the log files. If empty, log files are stored in the dot directories.")
+	flag.IntVar(&numberOfShards, "shard-number", 1, "The number of shards to create. The first created is assumed root.")
```

Comment:
there is some logic to pass `--shard-*` flags to the shards. Looks like we need a different name here?

Reply:
10 lines below actually :D

Reply:
passing `-shard-number` (single dash) will work but I can change to something else

Reply:
right now `Everything after "--" is considered a shard flag.`